Neos.Media/Classes/Domain/Model/ThumbnailGenerator/ImageThumbnailGenerator.php

```diff
@@ -80,4 +81,14 @@ public function refresh(Thumbnail $thumbnail)
             throw new Exception\NoThumbnailAvailableException($message, 1433109654, $exception);
         }
     }
+
+    /**
+     * @param PersistentResource $resource
+     * @param array $adjustments
+     * @return array
+     */
+    protected function processResource(PersistentResource $resource, array $adjustments)
+    {
+        return $this->imageService->processImage($resource, $adjustments);
+    }
```

Comment:
well I'd still do it that way instead of providing a different thumbnail converter since it's basically a add-on needed after the fact.. anyway that's irrelevant for this change though.

okay so to extend the thumbnail generator but change the generator, I see, still the method by itself it a little odd